setup.py

```diff
@@ -49,6 +49,7 @@ def run(self):
               'demo/toothpick.jpg',
               'units/citrine_en.txt',
               'units/constants_en.txt'
+              'units/tests/test_units.txt'
```

Comment:
Do we need to include this in the package?  The tests sources aren't included (I don't think), so the file is not referenced by anything in the package (I think).

Reply:
When I neglected to include it in the package, Jenkins failed this test because it couldn't find the file.

Reply:
Ahh, yeah, and I don't see a test_package option or anything.  :+1: